x-pack/plugins/fleet/common/types/models/agent.ts

```diff
@@ -36,8 +36,7 @@ export type AgentActionType =
   | 'UNENROLL'
   | 'UPGRADE'
   | 'SETTINGS'
-  // INTERNAL* actions are mean to interupt long polling calls these actions will not be distributed to the agent
-  | 'INTERNAL_POLICY_REASSIGN';
+  | 'POLICY_REASSIGN';
```

Comment:
I forgot to mention, do we have any integration tests for these agent actions that would need to get updated to cover this?

Reply:
it's well covered by the e2e tests